app/views/users/show.haml

```diff
@@ -1,4 +1,10 @@
 - content_for :title do @player.username end
+
+- if @player.username == "BrettOgden"
```

Comment:
Is there a reason we need to define this specifically? Either make a “party button” or we can just find people with cake days. 

Maybe when rendering names, if it’s somones cake day they get a cake emoji next to their name (similar to a tag).